eslint.config.mjs

```diff
@@ -15,7 +15,6 @@ const config = createConfig([
       sourceType: 'module',
       parserOptions: {
         tsconfigRootDir: import.meta.dirname,
-        project: ['./tsconfig.json'],
```

Comment:
I am guessing this was deleted just because it's not needed? Or was it causing a problem?